features/FEATURE_BLE/ble/gap/Types.h

```diff
@@ -48,7 +48,8 @@ typedef Duration<uint32_t, 625, Range<0x20, 0xFFFFFF> > adv_interval_t;
  * value 0x0000 means the advertising process never ends; it is accessible with
  * adv_duration_t::forever().
  */
-typedef Duration<uint16_t, 10000, Range<0x00, 0xFFFF>, /* forever */ Value<uint16_t, 0x0000> > adv_duration_t;
+// typedef Duration<uint16_t, 10000, Range<0x00, 0xFFFF>, /* forever */ Value<uint16_t, 0x0000> > adv_duration_t;
+typedef Duration<uint16_t, 10000, Range<0x00, 0xFFFF> > adv_duration_t;
```

Comment:
Did you mean to leave this change in? Just for testing?

Reply:
No. I was testing...I have [removed it](https://github.com/ARMmbed/mbed-os/compare/705752e1609b039eeb4fe9576cc6fe20d97502e1..244c974085576de2f25a69c81ea0c816bc4c22b0).